src/main/java/dev/emi/trinkets/TrinketModifiers.java

```diff
@@ -0,0 +1,43 @@
+package dev.emi.trinkets;
+
+import com.google.common.collect.Multimap;
+import dev.emi.trinkets.api.*;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.attribute.EntityAttribute;
+import net.minecraft.entity.attribute.EntityAttributeModifier;
+import net.minecraft.item.ItemStack;
+import net.minecraft.registry.entry.RegistryEntry;
+import net.minecraft.util.Identifier;
+
+public class TrinketModifiers {
```

Comment:
This class is indented using spaces, please use tabs

Reply:
ok